eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,801 @@
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/progressed",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/courseTerminated",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning/v1.0",
+        "prefLabel": {
+          "en": "Course terminated"
+        },
+        "definition": {
+          "en": "Learner has terminated the course."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/terminated",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/activityProgressed",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning/v1.0",
+        "prefLabel": {
+          "en": "Activity progressed"
+        },
+        "definition": {
+          "en": "Learner has progressed through the activity."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/progressed",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.name",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.type",
+            "presence": "included",
+            "scopeNote": {
+              "en-us": "This property should describe what activity type this statement goes with."
+            }
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/feedbackCommitted",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning/v1.0",
+        "prefLabel": {
+          "en": "Feedback committed"
+        },
+        "definition": {
+          "en": "Learner has committed feedback."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/answered",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
```

Comment:
If you want, you could move these common elements (like requiring mbox use) to a statement template without any determining properties (verb etc), which would cause it to apply to all statements. Then just leave that template off the alternates list in patterns.